app/views/Partners/PartnersOverview.tsx

```diff
@@ -111,14 +113,14 @@ const styles = StyleSheet.create({
     paddingHorizontal: 24,
   },
   bottomSheet: {
-    backgroundColor: colors.WHITE,
+    backgroundColor: Colors.WHITE,
```

Comment:
```suggestion
    backgroundColor: Colors.primaryBackground,
```
What do you think about using the design system based colors?  What happens when we reskin the app in the future? if we establish the pattern of named by intent rather than detail, we will save ourselves a lot of time.

Reply:
💯  big agree